class.jetpack.php

```diff
@@ -657,6 +657,7 @@ private function __construct() {
 		 * They check for external files or plugins, so they need to run as late as possible.
 		 */
 		add_action( 'wp_head', array( $this, 'check_open_graph' ),       1 );
+		add_action( 'amp_story_head', array( $this, 'check_open_graph' ),1 );
```

Comment:
```suggestion
		add_action( 'amp_story_head', array( $this, 'check_open_graph' ), 1 );
```

Reply:
Oops.

But I just realized that if this is applied, then the result would not be aligned:

```php
		add_action( 'wp_head', array( $this, 'check_open_graph' ),       1 );
		add_action( 'amp_story_head', array( $this, 'check_open_graph' ), 1 );
		add_action( 'plugins_loaded', array( $this, 'check_twitter_tags' ),     999 );
		add_action( 'plugins_loaded', array( $this, 'check_rest_api_compat' ), 1000 );
```

So I just went ahead and removed the alignment: 1ee745d76